### PR TITLE
replace set_pair_value by set_textarea_value

### PR DESCRIPTION
**Pull request by xiaocwan**
This is only for 4.0
@yapei Please review, thanks.

### CODE REVIEW

File: lib/rules/web/admin_console/4.0/secrets.xyaml [hunk 1]
```diff
@@ -40,8 +40,8 @@ create_generic_secret:
     if_param: key 
     ref: set_pair_key
   action:
-    if_param: value
-    ref: set_pair_value
+    if_param: textarea_value
+    ref: set_textarea_value
```
Comment (yapei) [inline lib/rules/web/admin_console/4.0/secrets.xyaml]: To not break your tests for `OCP-20800`, you also need update `set_textarea_value` rules to always send keys to the last `textarea`

Comment (xiaocwan) [reply lib/rules/web/admin_console/4.0/secrets.xyaml]: @yapei Never mind, I checked the features and OCP-20800 is the only case needs update, also there is another rule `click_one_dropdown_action` updating a param. So soner or later the OCP-20800 needs to patch a fix PR, so I update this one together this time, don't worry there's no big change.

Comment (yapei) [reply lib/rules/web/admin_console/4.0/secrets.xyaml]: ok for me

Comment (akostadinov) [reply lib/rules/web/admin_console/4.0/secrets.xyaml]: @yapei so you want PR merged?

Comment (yapei) [reply lib/rules/web/admin_console/4.0/secrets.xyaml]: @akostadinov  no problem, let's merge it

**Pull request merged**